src/api_gen.jl

```diff
@@ -43,37 +101,28 @@ end
 
 NvimApiObject(c, e::Ext) = NvimApiObject{int(e.typecode)}(c, e.data)
 #Not really module-interface clean, I know...
-function MsgPack.pack{N}(s, o::NvimApiObject{N})
-    MsgPack.pack(s, Ext(N, o.hnd))
-end
-
-
-# FIXME: the elephant in the room (i.e. handle &encoding)
```

Comment:
:+1: to remove this comment. The general opinion seems to be that plugins and guis are free to only support `encoding=utf8`